sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1397,7 +1398,6 @@ def test_series_agg(self, agg_method):
         'median',
         'sem',
         'mad',
-        'skew',
```

Comment:
There are a few other places where this is copied, can you make the change there as well? That should fix the precommit failures

Alternatively, it could be nice to have the list of non parallelizable aggregations defined in one place in a constant, then just update it there.